src/mots/directory.py

```diff
@@ -171,6 +188,10 @@ def __init__(self, people, bmo_data: dict):
                 parsed_real_name = parse_real_name(bmo_datum["real_name"])
                 person["name"] = parsed_real_name["name"]
                 person["info"] = parsed_real_name["info"]
+            else:
+                person["name"] = person.get("name", "")
+                person["info"] = person.get("info", "")
+                person["nick"] = person.get("nick", "")
```

Comment:
Unrelated to this PR?

Reply:
Yes -- it's a bug fix that I uncovered while writing some of the documentation.